### ISSUE TITLE
FE4: No Holy Blood Ares?

### ISSUE THREAD

**Issue by koumal8** [lushen124/Universal-FE-Randomizer#253]
I think something weird happened to my Ares. According to the log (I haven't gotten around to his chapter yet) he is a Mountain Thief with no Holy Blood but a Mystletainn still in his inventory? I attach my log for proof.

https://www.dropbox.com/s/ysd2wn49znlttwp/FE4R.html?dl=0

**Comment by lushen124** [comment]
Yeah, that is interesting. I don't think there's anything special about Ares, except that he is the "representative" for Hezul, but this is only used for shuffling blood, not randomizing blood. If he does show up like the log says, then I'll look into it more. Just want to make sure this isn't just a logging issue.
